.circleci/config.yml

```
@@ -228,13 +228,13 @@ workflows:
             parameters:
               python-version: &python-versions ["3.8", "3.9", "3.10", "3.11", "3.12"]
               pip-constraints:
-                # test the min and max (as of Jan 2024) version ranges
+                # test the min and max (as of July 2024) version ranges
                 - dimod==0.10.9 scipy==1.6.0 numpy==1.20.0 networkx==2.4
-                - dimod~=0.12.0 scipy~=1.11.0 numpy~=1.26.0 networkx~=3.0
+                - dimod~=0.12.0 scipy~=1.0 numpy~=2.0 networkx~=3.0
```

Why test with `scipy~=1.0`? Seems like it doesn't support any of the Python versions.

```suggestion
                - dimod~=0.12.0 scipy~=1.11.0 numpy~=2.0 networkx~=3.0
```

---

`~1.0` will install any version compatible with `1.0`. So equivalent to `<2`

---

Oh, that's right.